app/controllers/answers/create.js

```diff
@@ -0,0 +1,43 @@
+import moment from 'moment';
+import {
+  DATE_TIME_FORMAT,
+  DEFAULT_ERROR_MESSAGE,
+} from '~/utils/backend/constants';
+import { createAnswerSchema } from '~/utils/backend/validators/answer';
+import { sanitizeHTML } from '~/utils/backend/sanitizer';
+import { db } from '~/utils/db.server';
+
+export const createAnswer = async (body) => {
+  const { error, value } = createAnswerSchema.validate(body);
+
+  if (error) {
+    return {
+      error: { message: DEFAULT_ERROR_MESSAGE, detail: error.message },
+    };
+  }
+
+  const { answered_by_employee_id, answered_question_id, ...rest } = value;
+
+  let answer = await db.Answers.create({
+    data: {
+      ...rest,
+      answer_text: sanitizeHTML(value.answer_text),
+      answer_date: moment.utc().format(DATE_TIME_FORMAT),
+      AnsweredBy: {
+        connect: {
+          employee_id: answered_by_employee_id,
+        },
+      },
+      Question: {
+        connect: {
+          question_id: answered_question_id,
+        },
+      },
```

Comment:
Maybe I am wrong but these could be replaced with the column name directly:

```
answered_by_employee_id: answered_by_employee_id,
answered_question_id: answered_question_id
```

I do not know what connect does under the hood but it looks like it works as an alternative. Not sure which one is better, but just letting you know just in case

Reply:
@albertcastaned In fact, I first tried with that, but it gave me an error 'Unknown arg...' or something like that, and I saw something similar in a forum (I lost the link). It's something about the model connections, since those arguments are foreign keys connecting to Users and Questions respectively.

Reply:
Just seeing this, good spot! I havent seen this problem but we should bring it to the board so nobody comes to the same problem, or add it to wiki https://github.com/wizeline/wize-q-remix/wiki/Database-and-Prisma :